packages/scripts/integration-testing/cwAddresslistTest.ts

```diff
@@ -0,0 +1,33 @@
+import { program } from "commander";
+import fs from "fs/promises";
+
+import { deployCwAddressList } from "./../network-setup/deployCwAddressList";
```

Comment:
using this import will also trigger the `main` func in `deployCwAddressList`

Reply:
Removed. 
We can test using deployCwAddressList with localnet